Documentation/Signoffs/SteeringSystem.md

```diff
@@ -8,8 +8,10 @@ the data into a local microcontroller which will interpret the data and relay it
 ### Specifications & Constraints
 - Must measure angle of rotation of the steering wheel
 - Must use the Nissan Leaf OEM steering wheel as the input
-- Angle Shall be no more than 960 degrees to the left and right (2 and 2/3 rotations)
+- Angle shall be no more than 960 degrees from the left boundary to the right boundary (2 and 2/3 rotations)
```

Comment:
Again, shouldn't this be no less? Why are you requiring no more measurement than this?

Reply:
The steering wheel is only able to turn 960 degrees total. If the measured angle was more than that, then we know the angle must be wrong. Therefore, the angle must be no more than 960 degrees from left to right.